.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,4 +23,5 @@ Please also add any of the following appropriate labels to the PR:
 -->
 # Release Note for Downstream PRs (will be copied)
 ```releasenote
+
```

Comment:
MD/parsers don't recognize this as a proper code block unless there's a newline in the empty release note. Even if it doesn't have the newline, it's fine and won't cause a weird release note to get added, it just won't parse as a proper code block with language `releasenote` annotation.